richtextfx/src/integrationTest/java/org/fxmisc/richtext/api/FxmlTester.java

```diff
@@ -0,0 +1,26 @@
+package org.fxmisc.richtext.api;
+
+import javafx.stage.Stage;
+import javafx.fxml.FXMLLoader;
+import org.fxmisc.richtext.RichTextFXTestBase;
+import org.junit.Test;
+
+public class FxmlTester extends RichTextFXTestBase {
+
+    @Override
+    public void start(Stage stage) throws Exception {
+        // Nothing needed here
+    }
+
+    @Test
+    public void test_fxml_construction_of_area()
+    {
+        Object obj = null;
+        // FxmlTest.fxml is located in resources folder:
+        // src/integrationTest/resources/org/fxmisc/richtext/api/
+        try	{ obj = FXMLLoader.load( getClass().getResource( "FxmlTest.fxml" ) ); }
+        catch ( Exception EX ) { EX.printStackTrace(); }
+        org.junit.Assert.assertNotNull( obj );

```

Comment:
We should assert that the properties values are what we specified them to be.

Reply:
If FXMLLoader cannot set a property it will throw an exception, so if no exception is thrown then it's **guaranteed** to have been set :-)

Reply:
Good to know. Mind writing that in as a comment for those of us who don't use it that often?